### DISCUSSION TITLE
which file the data and index store

### DISCUSSION THREAD

**Discussion by huangxueliang** [milvus-io/milvus#37643] [Show and tell]
i want to known which file the data and the index store?
the minio just only two directory:
![image](https://github.com/user-attachments/assets/709e003d-515d-4cc2-8633-7433db144d8c)

**Comment by yhmo** [comment]
If you have called collection.insert() and the data has been flushed to disk, you will see "insert_log", vectors and scalar data is stored here.
"stats_log" stores the statistical information of segments, including bloom filters.
If you have called collection.create_index(), you will see "index_files", index is stored here.
If you have called collection.delete(), you will see "delta_log", this folder stores the primary key of deleted entities.

![Screenshot from 2024-11-13 14-07-57](https://github.com/user-attachments/assets/dbc47f75-3aa9-4d6c-82f1-3630561e5fbd)

**Reply by huangxueliang** [reply]
![image](https://github.com/user-attachments/assets/236a776b-bed1-42be-a3f1-c75c8c8937cd)

why there is no index_files? my collections have created index.

**Reply by yhmo** [reply]
Insert 1 million entities then you will see the folder.

**Reply by huangxueliang** [reply]
![image](https://github.com/user-attachments/assets/82b86077-1377-430d-8f0e-1d1e933c0bf5)
what means the number as marked,such as "451860017338870801"

**Reply by yhmo** [reply]
/insert_log/[collection ID]/[partition ID]/[segment ID]/[field ID]
/stats_log/[collection ID]/[partition ID]/[segment ID]/[field ID]

![Screenshot from 2024-11-13 16-46-38](https://github.com/user-attachments/assets/99ab1303-f695-4da7-ad6a-950e6e6dba02)

**Reply by huangxueliang** [reply]
how can i find the "collection ID" about a collection?

**Reply by yhmo** [reply]
Collection/partition/segment ID is for internal usage, users no need to know them.
You can use `print(collection.describe())` to print the detailed information of a collection, including its ID. But in pymilvus there are no interfaces to know a partition's id and a segment's ID.

**Reply by xiaofan-luan** [reply]
> how can i find the "collection ID" about a collection?

We are just gonna to have a new webUI which could show all the collection info there
